examples/dead-mans-switch-nft/index.rsh

```diff
@@ -0,0 +1,74 @@
+'reach 0.1';
+'use strict';
+
+export const main = Reach.App(() => {
+  const Creator = Participant('Creator', {
+    transferTo: Fun([], Address),
+  });
+  const Owner = ParticipantClass('Owner', {
+    notifyReceive: Fun([], Null),
+    notifyRenewNeeded: Fun([], Null),
+    notifyRenewSuccess: Fun([], Null),
+    notifyRenewFail: Fun([], Null),
+  });
+  const vNFT = View('NFT', { owner: Address });
+
+  init();
+
+  Creator.publish();
+
+  // const ownerOnly = (currentOwner, fn) =>
+  //   Owner.only(() => { if (this == currentOwner) { fn(interact); } });
+
+  var owner = Creator;
+  invariant(true);
+  while (true) {
+    vNFT.owner.set(owner);
+
+    if (owner == Creator) {
```

Comment:
This doesn't mean what you think it means... the way you have it the creator can never be a "normal" owner. You should not have any `if` like this in the program